### ISSUE TITLE
downloaded files are given .html file extension

### ISSUE THREAD

**Issue by fiendish** [kids-first/kf-ui-data-tracker#190]
Clicking the download links on the files at https://kf-ui-data-tracker.kidsfirstdrc.org/study/SD_7NQ9151J/files downloads files that end in .html. If you strip the .html suffix, then the file contents appear to be correct.

**Comment by dankolbman** [comment]
I don't get the extra extension on either of the files in that study. Does that happen if you download from other studies?

**Comment by fiendish** [comment]
Yup. AnethesiaLLC.pdf (SD_9PYZAHHE, SF_52E0MA6H) becomes AnethesiaLLC.pdf.html

**Comment by fiendish** [comment]
Are you setting the Content-Disposition header?

**Comment by dankolbman** [comment]
Ah I guess you must be using safari. I couldn't replicate in chrome or firefox, but it does happen in safari.

**Comment by dankolbman** [comment]
We must be sending the wrong Content-Disposition headers from the API per https://stackoverflow.com/questions/11487365/http-header-for-downloading-microsoft-word-and-excel-files.
Will move this to the api repo.
